cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -95,6 +95,9 @@ function start-kube-apiserver {
   if [[ -n "${TLS_CIPHER_SUITES:-}" ]]; then
     params+=" --tls-cipher-suites=${TLS_CIPHER_SUITES}"
   fi
+  if [[ -e "${KUBE_HOME}/bin/gke-internal-configure-helper.sh" ]]; then
+    gke-kube-apiserver-internal-sni-param params
```

Comment:
modifying params by reference is unexpected and hard to follow

Reply:
I saw on line 75-76. configure-etcd-params made the same choice. Any recommended alternatives?

Reply:
Would returning a string from the function, and concatenating params here better?

Reply:
that would certainly make it clearer from here that params were being modified

Reply:
Updated to use explicit returned string.